packages/web-react/src/components/Drawer/useDrawerStyleProps.ts

```diff
@@ -0,0 +1,44 @@
+import classNames from 'classnames';
+import { useClassNamePrefix } from '../../hooks';
+import { DrawerAlignmentXType } from '../../types';
+import { DRAWER_ALIGNMENT_DEFAULT } from './constants';
+
+export interface UseDrawerStylesProps {
+  drawerAlignment?: DrawerAlignmentXType;
+}
+
+export interface UseDrawerStylesReturn {
+  /** className props */
+  classProps: {
+    root: string;
+    panel: string;
+    content: string;
+    drawerCloseButton: string;
+  };
+}
+
+export const useDrawerStyleProps = (
+  { drawerAlignment }: UseDrawerStylesProps = {
+    drawerAlignment: DRAWER_ALIGNMENT_DEFAULT,
+  },
+): UseDrawerStylesReturn => {
+  const drawerClass = useClassNamePrefix('Drawer');
+  const drawerAlignClasses: Record<DrawerAlignmentXType, string> = {
+    left: `${drawerClass}--left`,
+    right: `${drawerClass}--right`,
+  };
+  const drawerPanelClass = `${drawerClass}Panel`;
+  const drawerContentClass = `${drawerPanelClass}__content`;
+  const drawerCloseButtonClass = `${drawerClass}CloseButton`;
+
+  const classProps = {
+    root: classNames(drawerClass, drawerAlignment && { [drawerAlignClasses[drawerAlignment]]: drawerAlignment }),
+    panel: classNames(drawerPanelClass, {}),
+    content: classNames(drawerContentClass, {}),
+    drawerCloseButton: drawerCloseButtonClass,
```

Comment:
```suggestion
    closeButton: drawerCloseButtonClass,
```
?
There is no drawer mention in other keys